Documentation/Maintenance/Release.md

```diff
@@ -997,7 +997,7 @@ excellent packaging.
 [ITK Software Guide]: https://itk.org/ItkSoftwareGuide.pdf
 [ITK wiki]: https://insightsoftwareconsortium.github.io/ITKWikiArchive/Wiki/ITK
 [ITK Sphinx examples]: https://examples.itk.org/
-[ITKReleases NumFOCUS GDrive]: https://drive.google.com/drive/folders/1-uE7lKAZN4PT52vJJuDqr6KT656r6Dn_?usp=drive_link
+[ITKReleases NumFOCUS GDrive]: https://drive.google.com/drive/u/2/folders/0AKRok0KBkv02Uk9PVA
```

Comment:
/u/2/ in the new address picks a specific Google profile for accessing. Preferably, we should leave this as the default.

Reply:
Also, the new address does not seem to be publicly available.

Reply:
Follow-up in #5403